docs/pages/server-access/guides/bpf-session-recording.mdx

```diff
@@ -57,19 +48,79 @@ $ uname -r
 </tbody>
 </table>
 
-## Step 1/3. Install and configure Teleport node
+(!docs/pages/includes/tctl.mdx!)
+
+## Background
```

Comment:
It feels like this section is too far down on the page. Most readers aren't going to know by default what "Enhanced Session Recording" is or why they would want it.

Should we move some or all of this to the intro before the pre-requisites so we don't lose readers?